pkg/controller/template/template_controller_test.go

```diff
@@ -475,7 +487,7 @@ func TestUpdateMachineConfig(t *testing.T) {
 		f.objects = append(f.objects, mcs[idx])
 	}
 
-	expmcs, err := getMachineConfigsForControllerConfig(templateDir, cc, []byte(`{"dummy": "dummy"}`), feat)
+	expmcs, err := getMachineConfigsForControllerConfig(templateDir, cc, []byte(`{"dummy": "dummy"}`), feat, nil)
```

Comment:
We should get a test here that storage object is populated and passed correctly.

Reply:
sounds like a good upgrade, i'll take a look